seedDB/README.md

```diff
@@ -13,7 +32,7 @@ This schema is required for cBioPortal release versions:
 When using a release version **> 2.0.0**, a migration step to a new database schema might be required. The migration process is described [here](https://github.com/cBioPortal/cbioportal/blob/master/docs/Updating-your-cBioPortal-installation.md#running-the-migration-script).
 
 **Schema 2.7.3**: [SQL file with create table statements](https://raw.githubusercontent.com/cBioPortal/cbioportal/v2.0.0/db-scripts/src/main/resources/cgds.sql)<br>
-**Seed database**: [seed-cbioportal_hg19_v2.7.3.sql.gz](https://github.com/cBioPortal/datahub/raw/master/seedDB/seed-cbioportal_hg19_v2.7.3.sql.gz)<br>
+**Seed database**: [seed-cbioportal_hg19_v2.7.3.sql.gz](https://github.com/cBioPortal/datahub/raw/175e5bb63119b517953c690684b7616bcb0102d9/seedDB/seed-cbioportal_hg19_v2.7.3.sql.gz)<br>
```

Comment:
@yichaoS could be ok to leave this at master? Once merged to master the link would be correct

Reply:
I suggested using a constant changeset hash. If in the future someone updates the seed database in master without updating the documentation, the content linked to would change, but the documentation would still preset the seed database as at schema 2.8.2. Also, when @yichaoS was trying to update the seed database link she needed to search back and find the commit hash for the previous (2.7.3) version in order to put in place a correct link for this "older version" section. By putting in the constant changeset hash link now we save the effort of the future seed DB updater so they do not need to  search for the appropriate hashcode.

Reply:
@sheridancbio 👌 makes sense